src/NDS.cpp

```diff
@@ -2073,11 +2073,6 @@ u32 ARM9Read32(u32 addr)
             return Slot2Cart_MemExpansionPak::ReadMemPak32(addr);
         }
         
-        if (addr == 0x80000AC)
-        {
-        		printf("ARM9 Read16: 0x%08x\n", ARM9->R[15]);
```

Comment:
Why is this removed?

Reply:
That was unnecessary logic used to debug my implementation.

Reply:
Okay, then I guess it's best to remove it from the commit that introduced it via interactive rebase (like other minor formatting issues).